Docs 923: Document `csi` camera module for the Jetson Orin Nano's embedded camera (#1716)
viamrobotics/docs: docs/components/board/jetson.md

Follow one of our Jetson [setup guides](/installation/) to prepare your board for running `viam-server` before configuring a `jetson` board.

> If after configuring a `jetson` board you want to add your Orin Nano's CSI camera to your robot, you can with Viam's `csi` module.

Author: ```suggestion
If after configuring a `jetson` board you want to add your Orin Nano's CSI camera to your robot, you can with Viam's `viam:camera:csi` model.
```

> Follow [these instructions](/extend/modular-resources/examples/csi) to do so.

Author: ```suggestion
Follow [these instructions](/extend/modular-resources/examples/csi/) to do so.
```

> If after configuring a `jetson` board you want to add your Orin Nano's CSI camera to your robot, you can with Viam's `viam:camera:csi` model.
> Follow [these instructions](/extend/modular-resources/examples/csi/) to do so.

Reviewer: Does it come with one by default? Otherwise this may be more appropriate:

```suggestion
If you have a CSI camera, follow [these instructions](/extend/modular-resources/examples/csi/) to configure it using the `viam:camera:csi` model.
```

Author: Paging @seanavery for confirmation but I assumed it did because the camera was embedded? I could be confused

Reviewer 2: CSI cameras must be purchased separately from the devboard. If it is helpful we can provide a list of commonly used CSI cameras for Jetson Orin-nano platform.
